static_src/components/navbar.jsx

```diff
@@ -129,7 +128,7 @@ export class Nav extends React.Component {
                   </ul>
                 </li>
                 <li className={ this.styler('marketplace') }>
-                  <a href={ this.marketplaceHref(org) }>Marketplace</a>
+                  <a href={ this.marketplaceHref(org) }>{ org.name } marketplace</a>
```

Comment:
**note:** this element should get the active class `sidenav-active-dashboard` any time its page is visible